app/views/users/edit.html.slim

```diff
@@ -57,6 +57,43 @@
   = f.input :company_name, label: 'Company name'
   = f.input :company_info, label: 'Company info', hint: 'Where is the office, how much space can you provide, etc.'
   = f.input :bio, label: 'About you', hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
+  h3.page-header Communication
+  .checkbox
+    label
+      = f.check_box :opted_in_newsletter, checked: @user.opted_in_newsletter.present?
```

Comment:
I fear I don't really understand this approach 🙈 
Didn't you define `opted_in_newsletter` on the `user` model just before? So why do you need to duplicate the internals of the method again in the view? I thought the whole point in defining all those methods was also to be able to have the form be simple as:
```ruby
= f.check_box :opted_in_newsletter
```
☝️ no manually setting it to `checked`.

Or is this just a leftover before refacting it to the final approach you chose?

The same question of course applies to all the other fields as well.